src/devtools/main.ts

```diff
@@ -362,12 +367,35 @@ async function initialize(): Promise<void>{
     }
 }
 
-function handleMcdevSBCommand(){
+function handleMCDevSBCommand(){
     editorOutput.showOuputChannel();
 }
 
+function getMCDevRelativePathComponents(relativePath: string): DevToolsPathComponents {
+    const [
+        credentialName,
+        businessUnit,
+        metadataType,
+        ...keys
+    ]: string[] = relativePath.split("/");
+    return { credentialName, businessUnit, metadataType, keys };
+}
+
+function logUnsupportedMtdtTypeNotification(action: string, unsupportedMtdtTypes: string | string[]){
+    [unsupportedMtdtTypes]
+    .flat()
+    .forEach((metadataType: string) => {
+        log(
+            "info", 
+            `SFMC DevTools currently does not support ${action} for the metadata type: '${metadataType}'`
+        );
+    });
+    devtoolsContainers.modifyStatusBar("mcdev", "info");
+    editorInput.handleShowNotificationMessage("info", mainConfig.messages.unsupportedMetadataType, []);
```

Comment:
```suggestion
    editorInput.handleShowNotificationMessage("error", mainConfig.messages.unsupportedMetadataType, []);
```

argument could be made for "warning" but I believe that this should be stronger and hence lean towards "error"